package.json

```diff
@@ -159,10 +161,12 @@
     "leaflet": "^1.4.0",
     "lodash": "^4.17.14",
     "mathjs": "^9.0.0",
+    "messageformat": "^2.3.0",
```

Comment:
Why is this needed?

Reply:
for pluralization.

Reply:
`messageformat` is the library that angular-translate uses under the hood for pluralization. So, it seemed reasonable to me use the same for pluralisation with ngx translate.

Reply:
messageFormat is a dependency of angular-translate. And angular-translate is our direct dependency. angular-translate is still used in some parts of the codebase. I think this library cannot be removed yet.

@srijanreddy98 WDYT?